packages/@okta/vuepress-site/docs/release-notes/index.md

```diff
@@ -13,9 +13,14 @@ The release notes are organized by year in descending order. In the left-side na
 
 We release first to preview orgs and then production orgs. Dates for preview releases are the earliest possible release date. Always check your org to verify the release for your org.
 
-To verify the current release for an org, check the footer of the Admin Console. If necessary, click **Admin** to navigate to your Admin Console.<br>
+To verify the current release and Okta solution for an org (Identity Engine or Classic), check the footer of the Admin Console. The version number is appended with E for Identity Engine orgs and C for Classic Engine orgs. If necessary, click **Admin** to navigate to your Admin Console. <br>
```

Comment:
```suggestion
To verify the current release and Okta solution for an org (Identity Engine or Classic), check the footer of the Admin Console. The version number is appended with E for Identity Engine orgs and C for Classic Engine orgs.<br>
```
Sorry, I forgot to ask what does the last line refer to? I'm not sure where to click **Admin**? Maybe this last line is old?

Reply:
As per Slack convo, keeping it in for now.